hoomd/hpmc/pytest/test_external_user.py

```diff
@@ -175,7 +198,8 @@ def test_electric_field(device, orientations, charge, result,
 
     sim = simulation_factory(two_particle_snapshot_factory())
 
-    ext = hoomd.hpmc.external.user.CPPExternalPotential(code=electric_field)
+    ext = hoomd.hpmc.external.user.CPPExternalPotential(code=electric_field,
+                                                        param_array=[0])
```

Comment:
This code doesn't use `param_array`. It should remain unchanged and should continue to function.